.projenrc.ts

```diff
@@ -1589,17 +1589,20 @@ const cliInteg = configureProject(
       compilerOptions: {
         ...defaultTsOptions,
         esModuleInterop: false,
+        target: 'es2022',
+        lib: ['es2022', 'esnext.disposable', 'dom'],
+        module: 'NodeNext',
       },
       include: ['**/*.ts'],
       exclude: ['resources/**/*'],
     },
     jestOptions: jestOptionsForProject({
       jestConfig: {
         coverageThreshold: {
-          statements: 40,
-          lines: 40,
+          statements: 25,
+          lines: 25,
```

Comment:
Not loving this. Can you at least explain why these are lowered?

Reply:
Yeah. There's a bunch of new code that involves calling NPM and making symlinks on disk, that doesn't have unit tests because it's either awkward to test or introduces sources of network calls and latency that we don't really want in unit tests (imo). 

I figured since this is for the integ tests, it is basically tested as part of every PR already.